src/test/resources/test-schema.sql

```diff
@@ -0,0 +1,17 @@
+create table users
+(
+    id       int         not null auto_increment primary key,
```

Comment:
```suggestion
    id       int         primary key auto_increment ,
```
아주 소소하지만 primary key가 먼저고 not null은 불필요합니다.